src/thunderbots/software/util/parameter/constants.py

```diff
@@ -0,0 +1,83 @@
+#######################################################################
+#                              Constants                              #
+#######################################################################
+
+AUTOGEN_FAILURE_MSG = '\033[91m ======== cfg autogen failure ======== \033[0m'
+
+# the working directory is set to the current folder in CMakeLists
+PATH_TO_YAML = './config/'
+PATH_TO_AUTOGEN_CFG = './autogenerated/cfg/'
+PATH_TO_AUTOGEN_CPP = './autogenerated/'
+DYNAMIC_PARMETERS_HEADER = 'dynamic_parameters.h'
+DYNAMIC_PARMETERS_CPP = 'dynamic_parameters.cpp'
+
+################
+# CPP Contants #
+################
+
+H_PARAMETER_DECL = 'extern Parameter<{type}> {name}; \n'
+CPP_PARAMETER_INSTNACE = 'Parameter<{type}> {name}(\"{path}\", {quote}{default}{quote});\n'
+
+NAMESPACE_OPEN = 'namespace {name} {{ \n'
+NAMESPACE_CLOSE = '}\n'
+
+AUTOGEN_WARNING = "// THIS FILE IS AUTOGENERATED, ANY CHANGES WILLBE LOST\n"
```

Comment:
Could we make this a bit more aggressive? Maybe something like:
```
AUTOGEN_WARNING = \
"""
/**
 *  !! WARNING !!
 *
 *  THIS FILE IS AUTOGENERATED, ANY CHANGES MADE WILL BE LOST
 *
 *  !! WARNING !!
 */
"""
``

Reply:
Fixed